sdks/python/apache_beam/testing/test_stream.py

```diff
@@ -188,7 +198,16 @@ def _infer_output_coder(self, input_type=None, input_coder=None):
   def expand(self, pbegin):
     assert isinstance(pbegin, pvalue.PBegin)
     self.pipeline = pbegin.pipeline
-    return pvalue.PCollection(self.pipeline, is_bounded=False)
+    if len(self.output_tags) == 0:
```

Comment:
nit: this can be simplified to just
```
if not self.output_tags:
```
Google's style guide recommends using boolean context whenever possible. Not sure whether there is such a rule in Beam. But since you're already using boolean context elsewhere in the code, it's good to be consistent.

Reply:
Gotcha, changed to ```if not self.output_tags```